### ISSUE TITLE
Unable to display image in chainlit.md

### ISSUE THREAD

**Issue by tituslhy** [Chainlit/chainlit#955]
**Describe the bug**
I'm trying to add images in the chainlit.md file but the typical markdown syntax fails. When you use image tag (like in HTML) you'll get the image tag back in text form. When you use [alt text](image file path or url) you'll get an empty image icon with the alt text. 

**Expected behavior**
I expected the image to render like it would on a GitHub README.MD. I did some digging of the source code and I think it might be because chainlit takes in text first and converts it to markdown.

**Comment by tpatel** [comment]
I was able to display an image from a `chainlit.md` file in the latest Chainlit release, I see two potential issues:

1) You might be missing an exclamation point.

> You can display an image by adding ! and wrapping the alt text in [ ]. Alt text is a short text equivalent of the information in the image. Then, wrap the link for the image in parentheses ().

[source](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images)

2) You can also try to replace a relative image url with an absolute image url (like https://placehold.co/600x400/png) to make sure this is not a problem with the image location.

Let me know if you still have an issue.

**Comment by snowyday** [comment]
I'm encountering an issue where I cannot display local images within my Markdown files. Here's a brief overview of the problem:
When using the Markdown Viewer in VSCode, the image appears. Here's the corresponding Markdown snippet:
<img width="1316" alt="Screenshot 2024-05-01 at 19 46 53" src="https://github.com/Chainlit/chainlit/assets/1930625/24e4e281-5a41-4fa8-b10f-e94f6bf83dde">

Similarly, when processed through Chainlint, the image does not render. The Markdown snippet used is identical:
<img width="693" alt="Screenshot 2024-05-01 at 19 47 34" src="https://github.com/Chainlit/chainlit/assets/1930625/952f1f78-ea55-460b-b849-787f773204e7">

**Comment by bluerose73** [comment]
I find a walkaround.
1. Create a ``public`` folder in the project root directory
2. Put images there
Chainlit can display images in the ``public`` folder.
